deploy/packer/app.json

```diff
@@ -79,5 +79,12 @@
         "Base_AMI":         "{{user `pvSourceAMI`}}"
       }
     }
+  ],
+  "post-processors": [
```

Comment:
I didn't know about this.  I'll have to read up.  pretty cool!

also, `packer build` has a `-machine-readable` that might have helped too.